composer.json

```diff
@@ -6,7 +6,7 @@
         "php": "^7.1",
         "behat/behat": "^3.3",
         "psr/container": "^1.0",
-        "symfony/dependency-injection": "^2.7 || ^3.0 || ^4.0"
+        "symfony/dependency-injection": "^3.4 || ^4.3 || ^5.0"
```

Comment:
Let's bump to `^5` only for newer releases, since older versions will still work with previous symfony stacks

Reply:
Give me a sec

Reply:
Nono, I mean supporting **only** `^5` 😅